modules/ztp-pgt-config-best-practices.adoc

```diff
@@ -15,3 +15,7 @@ Consider the following best practices when customizing site configuration `{poli
 * `MachineConfig` CRs should be included as `extraManifests` in the `SiteConfig` CR so that they are applied during installation. This can reduce the overall time taken until the cluster is ready to deploy applications.
 
 * `{policy-gen-cr}` CRs should override the channel field to explicitly identify the desired version. This ensures that changes in the source CR during upgrades does not update the generated subscription.
+
+* The default setting for `policyDefaults.consolidateManifests` is `true`. This is the recommended setting for DU profile. Setting it to `false` could impact scale deployments.
+
+* The default setting for `policyDefaults.orderPolicies` is `false`. This is the recommended setting for DU profile.
```

Comment:
Why it is recommended? @irinamihai can you please shed some light here? Thanks

Reply:
added some additional information to explain what is required if you want to change it to `true`

Reply:
I think the intent is to explain why it should be false Instead of implying it could be set to true. It doesn't make sense to have ordering while we still use ztp-wave to derive the order of the policies in the generated CGU. Simply state that the order of the policy is automatically determined based on ztp-wave by TALM and additional ordering can create conflict and failure.